cumulus_library/module_allowlist.json

```diff
@@ -4,6 +4,8 @@
     "allowlist" : {
         "covid_symptom" : "cumulus_library_covid/covid_symptom",
         "data_metrics" : "cumulus_library_data_metrics",
+        "rxnorm" : "cumulus_library_umls/rxnorm",
```

Comment:
There's an opportunity here to drop the repeated subdir (and switch the pip package here to rxnorm)

Reply:
oh yes i spaced on this one.